### PR TITLE
Server side render table implemented

### PR DESCRIPTION
**Pull request by venkatramanan16**
Description:
Server side render table implemented

Reviewers:
@Sivakumar00

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,6 +106,11 @@
   },
   "dependencies": {
     "postcss": "^8.3.6",
-    "react-table": "^7.7.0"
+    "react-infinite-scroll-component": "^6.1.0",
+    "react-table": "^7.7.0",
+    "react-table-sticky": "^1.1.3",
```
Comment (Sivakumar00) [inline package.json:110]: is it necessary to use this library?

File: package.json [hunk 2]
```diff
@@ -102,6 +106,11 @@
   },
   "dependencies": {
     "postcss": "^8.3.6",
-    "react-table": "^7.7.0"
+    "react-infinite-scroll-component": "^6.1.0",
```
Comment (Sivakumar00) [inline package.json]: I think, this is not needed

File: src/components/Table/VirtualizedRowTable.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  disableSort?: boolean;
+}
+
+const VirtualizedRowTable = <T extends Record<string, unknown>>(
+  props: PropsWithChildren<TableProperties<T>>,
+): ReactElement => {
+  // Use the state and functions returned from useTable to build your UI
+  const { columns, data, resize, defaultColumn } = props;
+
+  const scrollbarWidth = () => {
```
Comment (Sivakumar00) [inline src/components/Table/VirtualizedRowTable.tsx]: move this utils folder

**Review by Sivakumar00 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (venkatramanan16) [reply package.json:110]: Yes for sticky header we need react-table-sticky library

**Review by Sivakumar00 — APPROVED** [review]

**Pull request merged**